### PR TITLE
chore(ci): Using latest Chrome and Firefox releases to perform CI tests

### LINKED ISSUE #3635: Travis-CI fails sometimes
```
  <SplitButton>
    ✔ should open the menu when dropdown button is clicked
    ✔ should not open the menu when other button is clicked
    ✔ should invoke onClick when SplitButton.Button is clicked (prop)
    ✖ should not invoke onClick when SplitButton.Toggle is clicked (prop)
    ✔ should not invoke onClick when SplitButton.Toggle is clicked (prop) 
  ✖ "after each" hook for "Should pass disabled to both buttons"
Finished in 6.264 secs / NaN secs @ 19:12:32 GMT+0000 (Coordinated Universal Time)
SUMMARY:
✔ 351 tests completed
✖ 2 tests failed
FAILED TESTS:
  <SplitButton>
    ✖ should not invoke onClick when SplitButton.Toggle is clicked (prop)
      Firefox 56.0.0 (Linux 0.0.0)
    uncaught exception: AssertionError: expected 'next' to equal 'prev' (:0)
  ✖ "after each" hook for "Should pass disabled to both buttons"
    Firefox 56.0.0 (Linux 0.0.0)
  console.error.expected is undefined
  ./test/index.js/<@webpack:///test/index.js:63 <- test/index.js:158200:7
```

Example: https://travis-ci.org/react-bootstrap/react-bootstrap/builds/518914585?utm_source=github_status&utm_medium=notification

### PR DESCRIPTION
**Pull request by mxschmitt**
Before:

| Browser | Version |
| --------- | -------- |
| Firefox   | 56 |
| Chrome | 62 |

After: 

| Browser | Version |
| --------- | -------- |
| Firefox   | 66 |
| Chrome | 73 |

Should fix #3635. I guess just a Firefox bug which is now fixed.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ sudo: false
 
 language: node_js
 node_js:
-  - "10"
+  - lts/*
```
Comment (taion) [inline .travis.yml]: this seems to end up as 10 anyway?

File: .travis.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ sudo: false
```
Comment (taion) [inline .travis.yml]: can you remove the `sudo: false` above?

File: .travis.yml (hunk 1 above)
Comment (mxschmitt) [reply .travis.yml]: yep, but so we don't have to bump it each time, once Node.js will release a new LTS version.

Comment (taion) [reply .travis.yml]: ah cool

**Pull request merged**